### DISCUSSION TITLE
Projects - archived items

### DISCUSSION THREAD

**Discussion by RomanSladkov** [community/community#10365] [Projects and Issues]
Good afternoon!
Can you tell me how to view archived drafts of items?

Thanks

**Comment by sureshjoshi** [comment]
I've got this same problem, the instructions here: https://docs.github.com/en/issues/organizing-your-work-with-project-boards/tracking-work-with-project-boards/archiving-cards-on-a-project-board don't appear to work, because that button doesn't seem to exist (on my boards, anyways)

Appears to be better tracked here: https://github.com/github/feedback/discussions/7501
